pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[build-system]
+requires = ["setuptools", "setuptools_scm"]
```

Comment:
do we have it already in `setup.cfg`? maybe we should remove it from there

Reply:
Yeah, I'm not sure the exact issue, but the way its installed through things like `pre-commit` isn't actually installing the setup requirements and it errors out. Will remove that section then and move it into pyproject.toml 👍